protocol/examples/query/greet_server/wrong_greet_server_invalid_client_endpoint_address.json

```diff
@@ -0,0 +1,10 @@
+{
+  "jsonrpc": "2.0",
+  "id": 7,
+  "method": "greet_server",
+  "params": {
+    "public_key": "J9fBzJV70Jk5c-i3277Uq4CmeL4t53WDfUghaK0HpeM=",
+    "client_address": "wss://popdemo.dedis.ch/client",

```

Comment:
I don't think this endpoint is invalid in the sense that the url has the correct shape to me. Urls on the default ports should be accepted I think, I see that they made the same choice on the `greet_lao`, is there a reason for it ? 

Though, by accepting both default and specified port it might make the regex a bit uglier

Reply:
I also wasn't (and still am not) sure about why it was the case for the greet_lao. @pierluca could you perhaps clarify that point ?

Reply:
I agree, default ports should be acceptable and I suppose in the greet_lao it was a question of convenience at the time, or the default port scenario might not have been spotted.

Thankfully marking something as optional in a regex only makes it marginally uglier 😅

Reply:
I added them as optional now but we're still working on the regex to enforce the number of digits in the port number.